jobs/JGFS_ATMOS_CYCLONE_TRACKER

```diff
@@ -73,7 +73,7 @@ export err=$?; err_chk
 #------------------------------------------------
 
 machine=${machine:-$(echo ${SITE})}
-${USHens_tracker}/extrkr_gfs.sh ${loopnum} ${cmodel} ${CDATE} ${pert} ${DATA}
+${USHens_tracker}/extrkr_gfs.sh ${loopnum} ${cmodel} ${CDATE} ${pert} "${DATA}"
```

Comment:
Why not double quotes around the other inputs?

Reply:
My best guess is that we run a differential linter; meaning if there were shellcheck errors before and after, they cancel out and don't get reported here.  If you ran a single pass of shellcheck, I would expect it to get flagged.